CONTRIBUTING.md

```diff
@@ -66,6 +66,12 @@ See
 [`tox.ini`](https://github.com/open-telemetry/opentelemetry-python/blob/master/tox.ini)
 for more detail on available tox commands.
 
+### Benchmarks
+
+Performance progression of benchmarks for packages distributed by OpenTelemetry Python can be viewed as a [graph of throughput vs commit history](https://open-telemetry.github.io/opentelemetry-python/benchmarks/index.html). From this page, you can download a JSON file with the performance results.
+
+Running the `tox` tests also runs the performance tests if any are available. Benchmarking tests are done with `pytest-benchmark` and they output a table with results to the console.
+
```

Comment:
An example of how to run a benchmark test locally would be great.

Reply:
There's no example to give because they run together with the examples above :) i.e. running `tox -e test-core-sdk` will run both the Unit Test and the Performance Tests :)

Reply:
@NathanielRN this will show the perf change on your own machine between runs, right?

Do the JSON files need to be added to .gitignore for this PR?

Reply:
I think that's a good idea! I add `*.json` to the `.gitignore`. Can easily become `**/tests/*.json` in the future if we want to commit `.json` files.